src/digitalmarketplace/components/footer/template.njk

```diff
@@ -0,0 +1,86 @@
+{% from "../../../govuk-frontend/components/footer/macro.njk" import govukFooter -%}
+{{ govukFooter({
+  navigation: [
+    {
+      title: 'Contact',
+      columns: 3,
+      items: [
+        {
+          href: '/help',
```

Comment:
Do we really need to use `url_for('external.help')` for the href?

https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/layouts/_footer_categories.html#L8